pgcontents/tests/test_synchronization.py

```diff
@@ -330,6 +337,10 @@ def check_call(kwargs, expect_files, expect_warning=False):
             expect_warning=False,
         )
 
+        # Dispose of all engines created during this test to prevent leaked
+        # database connections.
+        self.cleanup_pgcontents_managers(six.itervalues(managers))
```

Comment:
I would have probably just used `managers.values()` rather than bothering with `six`, but this is fine.